### ISSUE TITLE
Proceed with the Django Girls tutorial

### ISSUE THREAD

**Issue by webknjaz** [kpi-web-guild/django-girls-blog-serhii73#8]
Do a PR per each tutorial page. For simplicity of tracking progress, you may add markdown checkboxed list here.

**Comment by serhii73** [comment]
@webknjaz https://github.com/kpi-web-guild/django-girls-blog-serhii73/pull/21 - I passed the first page.
Did I understand correctly that I need a new branch for each page?
From what branch I need new branch? From master? Right?
I need to merge into master, and only then create a new branch?
Am I not able to create a parallel branch? Before I merge into master?

**Comment by webknjaz** [comment]
Sometimes you can do stuff in parallel. According to GitHub Flow (which you should check out), you have to create new feature branches from master and then sync your branches with mater using merge or rebase (but I strongly don't recommend you mixing these two techniques).
So you don't have to wait till merging some feature into master unless it depends on that code. It this case you could create new branch from dependent one, but after it's merged to master sync your code in newer branch with rebase or merge.

**Comment by serhii73** [comment]
- [ ]  django_start_project

- [ ]  django_models

- [ ]  django_admin

- [ ] deploy

- [ ] django_urls

- [ ] django_views

- [ ] html

- [ ] django_orm

- [ ] dynamic_data_in_templates

- [ ] django_templates

- [ ] css

- [ ] template_extending

- [ ] extend_your_application

- [ ] django_forms

**Comment by serhii73** [comment]
@webknjaz I finished the tutorial.
http://serhii73.pythonanywhere.com/
https://github.com/kpi-web-guild/django-girls-blog-serhii73/pull/31

**Comment by webknjaz** [comment]
Finishing tutorial was never the point. The main goal is to show you how to work with code/github
